docs/source/access_control.rst

```diff
@@ -59,7 +59,7 @@ API Security
 --------------------------
 
 Don't forget to secure your API!  While we provide these convenience
-methods in Angular, you need to protect your server APIs as well.
+methods in AngularJS, you need to protect your server APIs as well.
```

Comment:
I belive that going forward the Angular team has decided to just call it "Angular" and not "AngularJS"